charts/influxdb-enterprise/templates/data-statefulset.yaml

```diff
@@ -117,10 +117,11 @@ spec:
               protocol: UDP
           livenessProbe:
             httpGet:
-              path: /ping
+              path: {{ .Values.livenessProbe.path | default "/ping" }}
```

Comment:
This is surprising to me. What other path would be more useful as a liveness probe? And what are the implications of this path being different than the path on the readiness probe?

Reply:
This was more so for additional functionality. I can remove it!